packages/bpk-component-banner-alert/src/BpkBannerAlert-test.js

```diff
@@ -85,21 +85,39 @@ describe('BpkBannerAlert', () => {
 
   it('should render correctly with show set false', () => {
     const tree = renderer.create(
-      <BpkBannerAlert type={ALERT_TYPES.WARN} message={message} dismissable show={false} />,
+      <BpkBannerAlert
+        type={ALERT_TYPES.WARN}
+        message={message}
+        dismissable
+        dismissButtonLabel="Dismiss"
```

Comment:
`dismissButtonLabel` was missing, a warning was displayed in the console (and due to line length, I had to break it)